depth.py

```diff
@@ -8,13 +8,16 @@ def depth(filename):
 
     for path in g.paths:
         for segment in path.segment_names:
-            name = segment.name
+            name = int(segment.name)
             if name in depth_map:
                 depth_map[name] += 1
             else:
                 depth_map[name] = 1
-    print(depth_map)
+    sorted_depth_items = sorted(depth_map.items())
 
+    with open('python_output.txt', 'w') as f: 
+        for pair in sorted_depth_items:
+            f.write(f'{pair[0]} {pair[1]}\n')
```

Comment:
I would recommend just printing this to standard output: i.e., just use `print`, not `f.write` (and no need to hard-code an output filename). This will make it easier to use the various scripts for different purposes in the future.